canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -1369,9 +1369,21 @@ Cypress.Commands.add("verifyNotificationIconType", (type) => {
 });
 
 Cypress.Commands.add("verifyCanvasTransform", (movString) => {
-	cy.get("#canvas-div-0 .d3-canvas-group")
-		.invoke("attr", "transform")
-		.should("eq", movString);
+	// Verify the argument passed is a string
+	if (movString === String) {
+		cy.get("#canvas-div-0 .d3-canvas-group", { timeout: 10000 })
```

Comment:
Is the `timeout` needed here. I see from the docs that the default is 4000 (4 seconds) which should be ample time for anything to happen in Common Canvas. We don't set this timeout anywhere else in the Cypress tests so if it isn't needed please remove it.